interfaces/api/pipeline/IAsyncRelocalizationPipeline.h

```diff
@@ -67,8 +67,11 @@ typedef enum {
  * This class provides the interface to define an asynchronous relocalization processing pipeline.
  */
 
-class [[xpcf::clientUUID("91a569da-5695-11ec-bf63-0242ac130002")]] [[xpcf::serverUUID("95913e8e-5695-11ec-bf63-0242ac130002")]] IAsyncRelocalizationPipeline :
-    virtual public IPipeline {
+class [[xpcf::clientUUID("91a569da-5695-11ec-bf63-0242ac130002")]] [[xpcf::serverUUID("95913e8e-5695-11ec-bf63-0242ac130002")]]
+#ifndef DOXYGEN_SHOULD_SKIP_THIS // Doxygen does not support custom DSL
```

Comment:
Is it needed only here? No pb with `[[clientUUID]]`? For an incoming change I requested a feature in XPCF in order to simplify the task of disabling these wrappers macros around attributes.
If there are other places where this check is needed, it might be better to disable the attributes in the .pro file when needed, or when we run doxygen etc instead of having to add those guards everywhere.
For now keep it like this, but it might be better to avoid this (if possible).